core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.ts

```diff
@@ -69,15 +64,6 @@ export class AudioBarComponent {
     this.selectedLanguage = {
       value: this.audioTranslationLanguageService.getCurrentAudioLanguageCode()
     };
-
-    this.track = {
-      progress: (progressPercentage: number) => {
-        if (progressPercentage) {
-          this.audioPlayerService.setProgress(progressPercentage / 100);
-        }
-        return this.audioPlayerService.getProgress() * 100;
-      }
-    };
```

Comment:
Removed unused code.
Since the audio bar is working fine without these lines.
Deferring @ashutoshc8101 to cross-check once.
Thanks!

Reply:
Thanks! @Radesh-kumar 
I have cross-checked this. This can be removed.
@Radesh-kumar Can you please remove `setProgress` and `getProgress` functions defined in audio player service too?

Reply:
done!